src/app/components/selection/selection_toolbar.ui

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="SelectionToolbarWidget" parent="GtkBox">
+    <property name="hexpand">1</property>
+    <property name="visible">0</property>
+    <child>
+      <object class="GtkActionBar" id="action_bar">
+        <property name="hexpand">1</property>
+        <property name="revealed">1</property>
+        <child>
+          <object class="GtkBox">
+            <property name="valign">center</property>
+            <style>
+              <class name="linked"/>
+            </style>
+            <child>
+              <object class="GtkButton" id="move_up">
+                <property name="icon-name">go-up-symbolic</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkButton" id="move_down">
+                <property name="icon-name">go-down-symbolic</property>
+              </object>
+            </child>
+          </object>
+        </child>
+        <child type="end">
+          <object class="GtkButton" id="queue">
```

Comment:
Could [AdwButtonContent](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.ButtonContent.html) be used here?

Reply:
ooooh neat, did not know this was a thing. I guess I might have to update the bindings version, will have a look thanks!